cea/technologies/solar/photovoltaic_thermal.py

```diff
@@ -547,15 +547,37 @@ def calc_PVT_module(tilt_angle_deg, IAM_b_vector, IAM_d, I_direct_vector, I_diff
 
 # investment and maintenance costs
 
-def calc_Cinv_PVT(P_peak, gv):
+def calc_Cinv_PVT(PVT_peak_kW, locator, config, technology = 0):
```

Comment:
what is a `0` technology? where would you look that up? there has to be a more communicative way to specify this...